marquez_airflow/dag.py

```diff
@@ -193,6 +206,62 @@ def handle_callback(self, *args, **kwargs):
 
         return super().handle_callback(*args, **kwargs)
 
+    def _alternate_create_flow(self, step, ti):
+        inputs = None
+        if step.inputs:
+            self.register_datasets(step.inputs)
+            inputs = self._to_dataset_ids(step.inputs)
+        outputs = None
+        if step.outputs:
+            self.register_datasets(step.outputs)
+            outputs = self._to_dataset_ids(step.outputs)
+
+        ti_location = self._get_location(ti.task)
```

Comment:
I see this is where it is used. I think maybe you should use a name that will not conflict with something that might be used in TaskInstance now or in the future. (`task` seems a possible collision candidate)